js/src/spbc-settings.js

```diff
@@ -722,6 +722,10 @@ jQuery(document).ready(function() {
             $this.text($this.data('default-text'));
         }
     });
+
+    jQuery(document).on('click', '.spbct_notice-dismiss', function(e) {
+        this.parentElement.parentElement.setAttribute('style', 'display:none;');
```

Comment:
Need to add logic which will remember closing the banner and not appears it again for the current user.